.travis.yml

```diff
@@ -1,7 +1,7 @@
 language: php
 
 env:
-  EBANX_WC_PLUGIN_VERSION: 3.0.6
+  EBANX_WC_PLUGIN_VERSION: 3.0.7
```

Comment:
I think it would be better to rename to WC_VERSION since only our own docker container will read it.